packages/jsii-pacmak/bin/jsii-pacmak.ts

```diff
@@ -183,7 +183,8 @@ async function npmPack(packageDir: string, tmpdir: string): Promise<string> {
         args.push('--loglevel=verbose');
     }
     const out = await shell('npm', args, { cwd: tmpdir });
-    return path.resolve(tmpdir, out.trim());
+    const lines = out.trim().split(os.EOL);
```

Comment:
Add a wee comment explaining what you are doing here and why "take the last STDOUT line of npm pack, as it will contain the name of the tarball, and "npm pack" may also run scripts which will emit output to STDOUT"